### PR TITLE
Added all of my mods that were missing, fixed regex to correctly work…

### PR DESCRIPTION
**Pull request by The-Fireplace**
… with my new naming system, and grouped my mods together

### CODE REVIEW

**Review by Pyker — DISMISSED** [review]
Some stuff needs to be changed before merging

File: NEMP/mods.json [hunk 1]
```diff
@@ -3723,15 +3714,159 @@
             "name": "cooking-for-blockheads"
         }
     },
+    "AdobeBlocks": {
+        "function": "CheckCurse",
+        "active": true,
+        "curse": {
+            "regex": "AdobeBlocks(?:-Forge)?(?:-(?P<mc>.+?))?-(?P<version>[0-9]+(?:\.[0-9]+)+)",
```
Comment (Pyker) [inline NEMP/mods.json]: Those instances of `\.` (and the ones in the other definitions) need to be changed to `\\.` so the backslash is properly escaped inside the JSON

**Review by Pyker — CHANGES_REQUESTED** [review]
Just a minor mistake that needs to be fixed, the rest seems to look good

File: NEMP/mods.json [hunk 2]
```diff
@@ -4101,24 +4227,6 @@
...
-    "Caterpillar": {
-        "function": "CheckCurse",
-        "active": true,
-        "curse": {
-            "regex": "Caterpillar-(?P<version>.+)\\.jar",
-            "id": "237628",
-            "name": "caterpillar"
-        }
-    },
-    "UnLogicII": {
```
Comment (Pyker) [inline NEMP/mods.json:4113]: Why was UnLogic II completely removed?

**Review by Pyker — CHANGES_REQUESTED** [review]

File: NEMP/mods.json [hunk 3]
```diff
@@ -3723,15 +3714,159 @@
...
+    "Fluidity": {
+        "function": "CheckCurse",
+        "active": true,
+        "curse": {
+            "regex": "Fluidity(?:-Forge)?(?:-(?P<mc>.+?))?-(?P<version>[0-9]+(?:\.[0-9]+)+)",
+            "id": "241409",
+            "name": "fluidity"
+        }
+    },
+    "InGameAccountSwitcher": {
```
Comment (SinZ163) [inline NEMP/mods.json:3780]: Why was InGameAccountSwitcher moved?

Comment (Pyker) [reply NEMP/mods.json:3780]: "(...) and grouped my mods together"

File: NEMP/mods.json (hunk 2 above)
Comment (The-Fireplace) [reply NEMP/mods.json:4113]: Pyker: UnLogic II is one of my mods, and has been replaced by Fire's Random Things. There is zero chance of it being updated by me, and nobody else should have any reason to update it.

Comment (The-Fireplace) [reply NEMP/mods.json:4113]: If having mods that will never be updated is a must, I can re-add it.

Comment (Pyker) [reply NEMP/mods.json:4113]: It isn't, I just wanted to make sure it wasn't a mistake from the reorganization.

**Review by Pyker — APPROVED** [review]

File: NEMP/mods.json (hunk 1 above)
Comment (Pyker) [reply NEMP/mods.json]: For some reason I'm struggling with GitHub PR reviews today... These changes are still a concern.

**Review by Pyker — CHANGES_REQUESTED** [review]

**Comment by The-Fireplace** [comment]
I know. I will fix it when I get a chance.

**Comment by The-Fireplace** [comment]
Done.

**Review by Pyker — APPROVED** [review]

**Comment by Pyker** [comment]
Thanks for your patience in this PR and I apologize for the review mess :)

**Comment by The-Fireplace** [comment]
Not a problem. I am glad I got quick feedback on how I could get my changes approved.

**Pull request merged**